src/BulletHell/PowerManager.java

```diff
@@ -0,0 +1,85 @@
+package BulletHell;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import BulletHell.Powerups.*;
+import edu.macalester.graphics.CanvasWindow;
+import edu.macalester.graphics.events.Key;
+
+/**
+ * A class responsible for adding the power boxes to the canvas, and binding the powers with their respective keys.
+ */
+public class PowerManager {
+    public static final int GAP = 50;
+
+    private List<Powerups> allPowerups = new ArrayList<>();
+    private List<Key> activateKeys = new ArrayList<>();
+    private CanvasWindow canvas;
+
+    /**
+     * Creates a manager for every game window. 
+     */
+    public PowerManager(CanvasWindow canvas, BulletHell bulletHell) {
+        this.canvas = canvas;
+
+        Slow slow = new Slow(bulletHell);
+        allPowerups.add(slow);
+        activateKeys.add(slow.getKey());
+
+        Eraser eraser = new Eraser(bulletHell);
+        allPowerups.add(eraser);
+        activateKeys.add(eraser.getKey());
+
+        Heal heal = new Heal(bulletHell);
+        allPowerups.add(heal);
+        activateKeys.add(heal.getKey());
```

Comment:
You could refactor out the repetition here: `addPowerUp(new Heal(bulletHell));` and make that method handle both `allPowerups` and the key binding.